README.md

```diff
@@ -1,2 +1,33 @@
 # emap-actions
-Reusable GitHub actions for EMAP
+Reusable GitHub actions for EMAP projects
+
+
+## Examples
+
+To tag a list of repositories
```

Comment:
worth describing why you'd want this

Reply:
Have updated appropriately – thoughts/suggestions/commits to make it clearer are appreciated!